qutebrowser/config/configtypes.py

```diff
@@ -2014,6 +2014,15 @@ def __init__(self, none_ok: bool = False) -> None:
             ('last', "At the end."))
 
 
+class LogLevel(String):
+
+    """Log level."""
+
+    def __init__(self, none_ok: bool = False) -> None:
+        super().__init__(none_ok=none_ok)
+        self.valid_values = ValidValues(*log.LOG_LEVELS.keys())
```

Comment:
```suggestion
        self.valid_values = ValidValues(*[level.lower()
                                        for level in log.LOG_LEVELS])
```

Because settings in the config usually are lower-case.